apps/packages/ui/src/assets/tailwind-shared.css

```
@@ -216,19 +237,22 @@ body {
   }
 
   .panel-card {
-    @apply rounded-xl border border-border bg-surface text-text shadow-card transition-shadow duration-150 ease-out motion-reduce:transition-none;
+    @apply rounded-xl border border-border bg-surface text-text shadow-card transition-shadow ease-out motion-reduce:transition-none;
+    transition-duration: var(--duration-base);
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)